project/profiles/capacity4more/capacity4more.info

```diff
@@ -58,7 +58,6 @@ dependencies[] = og_ui
 dependencies[] = og_purl
 dependencies[] = og_vocab
 dependencies[] = og_menu
```

Comment:
Yes, we need this contrib module for the customizable menus in projects. Our `c4m_og_menu` just changes some behavior.